firestore/spec/helpers.rb

```diff
@@ -0,0 +1,21 @@
+require "google/cloud/firestore"
+
+def delete_collection collection_name:
+  firestore = Google::Cloud::Firestore.new project_id: ENV["FIRESTORE_PROJECT_ID"]
```

Comment:
I missed this... set project_id as a parameter and pass it in when running the command in tests.